guides/common/modules/proc_installing-the-load-balancer.adoc

```diff
@@ -24,6 +24,9 @@ However, you can install any suitable load balancing software solution that supp
 ----
 . Configure the load balancer to balance the network load for the ports as described in xref:ports-configuration-for-the-load-balancer[].
 For example, to configure ports for HAProxy, edit the `/etc/haproxy/haproxy.cfg` file to correspond with the table.
+ifdef::satellite[]
+See the Red Hat Knowledgebase solution https://access.redhat.com/solutions/4062981[Configuration example for haproxy.cfg for HAProxy load balancer with Satellite 6] for an example configuration.
```

Comment:
I found this format used for other knowledgebase links, so you might want to use it for consistency. I don't think you need "for an example configuration" because the title is so specific. Suggestion:

"See the Knowledgebase article https://access.redhat.com/solutions/4062981[Configuration example for haproxy.cfg for HAProxy load balancer with Satellite 6] on the Red{nbsp}Hat Customer Portal."

Reply:
I tried to be over-cautious to address this link as an example.
I agree with your view & suggestion  :+1: 

Although at some places we use `Red{nbsp}Hat Customer Portal` while there is also `_Red{nbsp}Hat Knowledgebase_` at other places addressing the KCS link in the document. Should this be flexible or consistent?

@Lennonka can you also share your view?

Reply:
We should strive to be as consistent as possible, so I would stick with RH Knowledgebase specifically.
FYI: [Formatting of KCS links according to the RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links)

On the other things, I agree with Avital.

Reply:
Thanks, @Lennonka . I didn't know that that RHSSG covered Kbase links.

Reply:
Thanks :star: